React-Styled-Components/react-styled-components_box/pages/index.js

```diff
@@ -1,10 +1,19 @@
 import BoxWithClassName from "../components/BoxWithClassName/BoxWithClassName.js";
+import BoxWithStyledComponents from "../components/BoxWithStyledComponents.js";
+import styled from "styled-components";
 
 export default function HomePage() {
   return (
-    <div>
+    <StyledPage>
       <BoxWithClassName />
       <BoxWithClassName isBlack />
-    </div>
+      <BoxWithStyledComponents />
+      <BoxWithStyledComponents $isBlack />
```

Comment:
Here Jan told me the the $ is not needed, only when we are passing the argument to the styled component itself.